src/Orchard.Data/ServiceCollectionExtensions.cs

```diff
@@ -25,6 +27,8 @@ public static IServiceCollection AddDataAccess(this IServiceCollection services)
             services.TryAddDataProvider(name: "Sql Server", value: "SqlConnection", hasConnectionString: true);
             services.TryAddDataProvider(name: "Sql Lite", value: "Sqlite", hasConnectionString: false);
 
+            SqlDialectFactory.RegisterSqlDialect("diagnosticdbconnection", new SqlServerDialect());
```

Comment:
This is the only one at the moment. The new Glimpse stuff is very limited at the moment, so maybe we could get this stuff in, then I can iterate on it more whilst other people take a look.